parts/k8s/kuberneteswinagentresourcesvmas.t

```diff
@@ -207,9 +216,9 @@
       },
       "type": "Microsoft.Compute/virtualMachines"
     },
-    {{if UseManagedIdentity}}
+    {{if and UseManagedIdentity (not UserAssignedIDEnabled)}}
```

Comment:
The `not UserAssignedIDEnabled` condition is new w/ this change, is that intentional?

Reply:
Yes it was intentional as this condition was missing when `UserAssignedIDEnabled` feature was added awhile back.